assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.css

```diff
@@ -0,0 +1,185 @@
+/** -------------------------------------------------------------------------
+ * Event Espresso BiggieCalendarDate Stylesheet
+ * © 2020 Event Espresso
+ * ------------------------------------------------------------------------- */
+
+.ee-biggie-calendar-date-bg {
+	box-sizing: border-box;
+	color: var(--ee-color-white);
+	height: 100%;
+	padding: var(--ee-padding-tiny) var(--ee-padding-micro);
+	position: absolute;
+	text-align: center;
+	width: calc(var(--ee-padding-micro) * 34);
+	vertical-align: bottom;
+}
+
+.ee-biggie-calendar-date-bg.ee-bcd-pos-left {
+	float: left;
+	left: 0;
+}
+.ee-biggie-calendar-date-bg.ee-bcd-pos-right {
+	float: right;
+	right: 0;
+}
+
+.ee-biggie-calendar-date-header {
+	font-size: var(--ee-font-size-micro);
+	letter-spacing: var(--ee-letter-spacing-font-size-micro);
+	line-height: var(--ee-font-size-bigger);
+	margin: 0 0 var(--ee-margin-micro);
+}
+
+.ee-biggie-calendar-date .ee-bcd-weekday {
+	font-size: var(--ee-font-size-smaller);
+	font-weight: normal;
+	line-height: var(--ee-font-size-big);
+	letter-spacing: var(--ee-letter-spacing-font-size-tiny);
+}
+
+.ee-biggie-calendar-date .ee-bcd-month {
+	font-size: calc(var(--ee-font-size-tiny) * 2);
+	font-weight: bold;
+	line-height: calc(var(--ee-font-size-tiny) * 2);
+}
+
+.ee-bcd-month-day-sep {
+	display: none;
+}
+
+.ee-biggie-calendar-date .ee-bcd-day {
+	opacity: 0.8;
+	font-size: calc(var(--ee-font-size-big) * 2);
+	font-weight: bold;
+	line-height: calc(var(--ee-font-size-big) * 2);
+	text-shadow: var(--ee-text-shadow-inset);
+}
+
+.ee-biggie-calendar-date .ee-bcd-year {
+	font-size: var(--ee-font-size-tiny);
+	letter-spacing: calc(var(--ee-letter-spacing-font-size-micro) * 2);
+	line-height: var(--ee-font-size-default);
+	left: var(--ee-padding-nano);
+	position: relative;
+}
+
+.ee-biggie-calendar-date .ee-bcd-time {
+	font-size: var(--ee-font-size-smaller);
+}
+
+.ee-biggie-calendar-date-footer {
+	font-size: var(--ee-font-size-smaller);
+	line-height: var(--ee-font-size-bigger);
+	margin: var(--ee-margin-micro) 0 0;
+	text-shadow: var(--ee-text-shadow-inset);
+	white-space: pre-line;
+}
+
+.ee-biggie-calendar-date-footer .ee-status-tag {
+	display: block;
+	font-weight: bold;
+	letter-spacing: var(--ee-letter-spacing-font-size-micro);
+	text-transform: uppercase;
+}
+
+.ee-biggie-calendar-date-bg .ee-edit-calendar-date-btn {
+	color: rgba(255, 255, 255, 0.5);
+	display: inline-block;
+}
+
+.ee-biggie-calendar-date-bg .ee-edit-calendar-date-btn:focus:enabled,
+.ee-biggie-calendar-date-bg
+	.ee-edit-calendar-date-btn:not(:disabled):not([aria-disabled='true']):not(.is-default):hover,
+.ee-biggie-calendar-date-bg
+	.ee-edit-calendar-date-btn:not(:disabled):not([aria-disabled='true']):not(.is-default):active {
+	background-color: transparent;
+	color: var(--ee-color-white);
+	box-shadow: none;
+}
```

Comment:
sass would come handy here, converted to scss it will look like this:
```
.ee-biggie-calendar-date-bg .ee-edit-calendar-date-btn {
  &:focus:enabled {
    background-color: transparent;
    color: var(--ee-color-white);
    box-shadow: none;
  }

  &:not(:disabled):not([aria-disabled='true']):not(.is-default) {
    &:hover, &:active {
      background-color: transparent;
      color: var(--ee-color-white);
      box-shadow: none;
    }
  }
}
```

Reply:
this part is a bit confusing `&:not(:disabled):not([aria-disabled='true']):not(.is-default)`

Reply:
cool... thnx for posting that, I'll have to start trying that out now that we have it enabled.

I don't remember the details behind those `:not` styles since it was a while ago when this was first written and I've just copied it over from EDTR v1. I _think_ that they are undoing some styles applied by WP core. May not be necessary now since we are not utilizing WP components anymore. Will revisit the styles when they get applied to actual EDTR elements.